ods_ci/libs/CaikitPythonClient.py

```diff
@@ -0,0 +1,55 @@
+from robot.api import logger
+from robot.libraries.BuiltIn import BuiltIn
+from robotlibcore import keyword
+from caikit_nlp_client import HttpClient, GrpcClient
+
+
+class CaikitPythonClient:
+    """keywords to test https://github.com/opendatahub-io/caikit-nlp-client"""
+    def __init__(self):
+        self.BuiltIn = BuiltIn()
+        self.self.client = None
+
+    @keyword
+    def get_http_client_without_ssl_validation(self, host, port):
+        self.self.client = HttpClient(f"https://{host}:{port}", verify=False)
```

Comment:
too many `self`?

Reply:
yeah I saw yday. i thought I already pushed the commit with the fix, but I didn't xD will do